content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -58,15 +58,17 @@ By default, a pod is non-isolated for egress; all outbound connections are allow
 A pod is isolated for egress if there is any NetworkPolicy that both selects the pod and has
 "Egress" in its `policyTypes`; we say that such a policy applies to the pod for egress.
 When a pod is isolated for egress, the only allowed connections from the pod are those allowed by
-the `egress` list of some NetworkPolicy that applies to the pod for egress.
+the `egress` list of some NetworkPolicy that applies to the pod for egress. Reply traffic for those
+allowed connections will also be implicitly allowed.
```

Comment:
Is this about reply packets for traffic allowed by an _ingress_ rule? We're talking here about which packets are allowed to egress.

Reply:
@sftim she means that when you do a connection to B and is allowed in egress, the reply from B, that belongs to that connection , will be allowed too 

so it will be a stateful firewall as it will implicitly recognize that the incoming traffic is allowed because belongs to a connection that was allowed in egress